etcd3/client.py

```diff
@@ -329,15 +329,19 @@ def get_all(self, sort_order=None, sort_target='key'):
             for kv in range_response.kvs:
                 yield (kv.value, KVMetadata(kv, range_response.header))
 
-    def _build_put_request(self, key, value, lease=None):
+    def _build_put_request(self, key, value, lease=None, prev_kv=None):
```

Comment:
Can you use `False` rather than `None` by default? That sounds more logical since it's a boolean.